src/components/Contacts/ContactFlow/ContactFlowDragLayer/ContactFlowDragLayer.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { CSSProperties, RefObject } from 'react';
 import { XYCoord, useDragLayer } from 'react-dnd';
 import theme from 'src/theme';
+import { DraggedContact } from '../ContactFlowRow/ContactFlowRow';
```

Comment:
When running locally I noticed that it looks like it is showing duplicate contacts when dragging. Does that have to do with using `DraggedContact`?

Reply:
I also noticed that some contacts are not draggable. I was using Pedro's account. Some of those contacts were in the Partner Care column. Beast and Dalmation

Reply:
> When running locally I noticed that it looks like it is showing duplicate contacts when dragging. Does that have to do with using `DraggedContact`?

I was seeing that sporadically on `main` when I was doing the autoscroll work, so I don't think it's because of task phases or this PR. We should address that later on though. I added an item to our sheet.

> I also noticed that some contacts are not draggable. I was using Pedro's account. Some of those contacts were in the Partner Care column. Beast and Dalmation.

Thanks for noticing that. I'll see if I can figure out what's happening.

Reply:
@caleballdrin I'm having trouble consistently reproducing both of those bugs. And when I can reproduce it, I'm not getting anywhere with debugging. Do you think it's OK to move on for now and come back to this after go live?

Reply:
Ok yes if it is unrelated then lets push it and fix it later.

Reply:
I added two items to our functionality review sheet for each of these two issues.